app/views/books/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= @book.title %>
+
+<%= form_for [@book, @review] do |f| %>
+  <%= f.label :name %>
+  <%= f.text_field :name %>
+  <%= f.label :body %>
+  <%= f.text_field :body %>
+  <%= f.submit %>
+<% end %>
```

Comment:
None of your user stories refer to creating a new records. I wouldn't expect to see a form.